packages/@expo/fingerprint/README.md

```diff
@@ -171,6 +196,45 @@ Here's how to use **.fingerprintignore**: To skip a whole folder but keep some f
 !/app/ios/Podfile.lock
 ```
 
+### **fingerprint.config.js**
+
+You can customize the fingerprinting behavior by creating a **fingerprint.config.js** file in your project root. This file allows you to specify custom configurations, such as skipping certain fingerprint sources, adding extra fingerprint sources, or enabling debug mode.
+
+Below is an example **fingerprint.config.js** configuration, assuming you have `@expo/fingerprint` installed as a direct dependency:
+
+```js
+const { SourceSkips } = require('@expo/fingerprint');
+
+/** @type {import('@expo/fingerprint').Config} */
+const config = {
+  sourceSkips:
+    SourceSkips.ExpoConfigRuntimeVersionIfString |
+    SourceSkips.ExpoConfigVersions |
+    SourceSkips.PackageJsonAndroidAndIosScriptsIfNotContainRun,
+};
+
+module.exports = config;
+```
+
+If you are using `@expo/fingerprint` through `expo-updates` (where `@expo/fingerprint` is installed as a transitive dependency), you can use the following example **fingerprint.config.js** configuration with dynamic `require`:
+
+```js
+const { createRequire } = require('module');
+const requireFromUpdates = createRequire(require.resolve('expo-updates'));
+const { SourceSkips } = requireFromUpdates('@expo/fingerprint');
```

Comment:
that's not ideal. maybe we should do some change in sdk 52 for either
  - enforce people who using fingerprint runtime policy to install fingerprint as project dependency. expo-updates would use resolved fingerprint from project.
  - re-export as `expo-updates/fingerprint`

Reply:
If all goes to plan, the fingerprint runtime policy will become the default, so keep that in mind when we decide how to do this. I defer to you though on what you think the best experience is though.

Reply:
good to know the context. let me merge this first.
i would go for the following plan:
1. support to pass strings as sourceSkips in fingerprint.config.js. then people don't have to import fingerprint in their config.
2. still re-export `expo-updates/fingerprint`, in case people want to leverage type annotation as `@type {import('expo-updates/fingerprint').Config}`